support-frontend/assets/pages/[countryGroupId]/oneTimeCheckout.tsx

```diff
@@ -470,8 +470,7 @@ function OneTimeCheckoutComponent({
 																	setRecaptchaToken(token);
 																}}
 																onRecaptchaExpired={() => {
-																	// no-op
-																	// ToDo - Should we not expire this?
+																	setRecaptchaToken(undefined);
```

Comment:
Should we do the same thing on the generic checkout?

Reply:
Have never used this before, maybe do a separate PR so can revert easily?